toolkit/imageconfigs/scripts/cleanup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eux
+
+# cleanup symlinks created by the toolkit that are not needed for base images
+if [ -L /srv ]; then
+  echo "Removing /srv symlink"
+  rm /srv
+else
+  echo "/srv symlink does not exist"
+fi
+
+# cleanup any logs that may have been created during the build
+if [ -L /var/log ]; then
+  echo "Clearing /var/log"
```

Comment:
Should this be "Removing /var/log/ symlink" too?

Reply:
see my other comment